test/cppsuite/tests/hs_cleanup.cxx

```diff
@@ -0,0 +1,58 @@
+/*-
```

Comment:
What is that new test for ?

Reply:
The original ticket work is to add this test, which will then have its update operation modified in a further ticket